src/sentry/digests/utils.py

```diff
@@ -132,6 +133,7 @@ def get_participants_by_event(
             target_identifier=target_identifier,
             event=event,
             fallthrough_choice=fallthrough_choice,
+            rules=rules,
```

Comment:
I think the only place that we weren't sending `rules` is for digests so I could rewrite [get_send_to](https://github.com/getsentry/sentry/blob/master/src/sentry/notifications/utils/participants.py#L391) a bit to expect it, though I have to update a bunch of tests as well. Might be better suited for a separate PR.

Reply:
one thing we should probably do when we re-do get_send_to is that right now we check if any of the rules have a snooze for all , then no notification gets sent which I don't think works for digests. It might be trickier because for digests I think we will have to change what gets sent in the notification, not just if it gets sent or not

Reply:
Ohhh yeah that's a good point ... that should probably be addressed now.